src/platform/linux/input.cpp

```diff
@@ -1510,7 +1510,7 @@ namespace platf {
     std::stringstream ss;
     ss << std::hex << std::setfill('0');
     for (const auto &ch : str) {
-      ss << ch;
+      ss << static_cast<uint_least32_t>(ch);
```

Comment:
>char32_t — type for UTF-32 character representation, required to be large enough to represent any UTF-32 code unit (32 bits). It has the same size, signedness, and alignment as [std::uint_least32_t](https://en.cppreference.com/w/cpp/types/integer), but is a distinct type.
https://en.cppreference.com/w/cpp/language/types#char32_t

Therefore we should expect no data loss with a static_cast to uint_least32_t